tests/dragonfly/replication_test.py

```diff
@@ -252,7 +252,7 @@ async def disconnect(replica, c_replica, crash_type):
 
     # Check master survived all disconnects
     assert await c_master.ping()
-
+    await c_master.close()
```

Comment:
all the replication tests have this problem not only this one.. and I believe there is bug in the aioredis lib version we use that close does not realy close and we need to use connection_pool.disconnect
I will open PR with this change